### DISCUSSION TITLE
blog/2023/01/09/syllabus-csl-pandoc/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#91] [Blog comments]
# blog/2023/01/09/syllabus-csl-pandoc/

By default, pandoc doesn’t include full bibliographic references inline in documents, but with one tweak to a CSL file, you can create syllabus-like lists of citations with full references

https://www.andrewheiss.com/blog/2023/01/09/syllabus-csl-pandoc/

**Comment by bbolker** [comment]
This is really helpful and **almost** works for me, but ... I want an *annotated* bibliography. `chicago-annotated-bibliography.csl` was doing what I wanted, but ... if I change "note" to "in-text" there, I get in-text references (but just author and title). If I use `chicago-fullnote` as in your example, I get the full refs but not the annotations ... I guess I can try to hack these together ...

**Comment by CarlColglazier** [comment]
Thanks for this. Exactly what I needed for my use-case with a clear presentation!

**Comment by dsanchezp18** [comment]
Thanks so much!

**Comment by jtkulas** [comment]
This is fantastic! Thank you so much for the resource - greatly appreciated!! :+1:
